src/constants/sfx.asm

```diff
@@ -2,278 +2,278 @@
-MUSIC_SILENCE                                   equ $FF ; Seems to try to silence the voice channels. Does not always correctly silence the music.
+MUSIC_NONE                              equ $00
+MUSIC_TITLE_SCREEN_INTRO                equ $01
+MUSIC_MINIGAME                          equ $02 ; Trendy Game, Fishing, Camera Shop
+MUSIC_GAME_OVER                         equ $03
+MUSIC_MABE_VILLAGE                      equ $04
+MUSIC_OVERWORLD                         equ $05
+MUSIC_TAL_TAL_RANGE                     equ $06
+MUSIC_SHOP                              equ $07 ; Town Shop, Witch's Hut
+MUSIC_RAFT_RIDE_RAPIDS                  equ $08
+MUSIC_MYSTERIOUS_FOREST                 equ $09
+MUSIC_TOWN_HOUSE                        equ $0A
+MUSIC_ANIMAL_VILLAGE                    equ $0B
+MUSIC_FAIRY_FOUNTAIN                    equ $0C
+MUSIC_TITLE_SCREEN                      equ $0D ; If intro skipped with Start
+MUSIC_BOWWOW_KIDNAPPED                  equ $0E
+MUSIC_OBTAIN_SWORD                      equ $0F
+MUSIC_OBTAIN_ITEM                       equ $10
+MUSIC_FILE_SELECT                       equ $11
+MUSIC_EGG_MAZE                          equ $12
+MUSIC_KANALET_CASTLE                    equ $13
+MUSIC_TAIL_CAVE                         equ $14
+MUSIC_BOTTLE_GROTTO                     equ $15
+MUSIC_KEY_CAVERN                        equ $16
+MUSIC_ANGLERS_TUNNEL                    equ $17
+MUSIC_AFTER_BOSS                        equ $18
+MUSIC_BOSS                              equ $19
+MUSIC_INTRO_CUTSCENE                    equ $1A
+MUSIC_OBTAIN_INSTRUMENT                 equ $1B
+MUSIC_INTRO_WAKE_UP                     equ $1C ; Marin & Tarin's House
+MUSIC_OVERWORLD_SWORDLESS               equ $1D
+MUSIC_DREAM_SHRINE_SLEEP                equ $1E
+MUSIC_SOUTHERN_SHRINE                   equ $1F
+MUSIC_FULL_MOON_CELLO                   equ $20
+MUSIC_2D_UNDERGROUND                    equ $21
+MUSIC_OWL                               equ $22
+MUSIC_FINAL_BOSS                        equ $23
+MUSIC_DREAM_SHRINE_BED                  equ $24
+MUSIC_HEART_CONTAINER                   equ $25
+MUSIC_CAVE                              equ $26
+MUSIC_OBTAIN_POWERUP                    equ $27
+MUSIC_CONCH_HORN                        equ $28
+MUSIC_SEA_LILYS_BELL                    equ $29
+MUSIC_SURF_HARP                         equ $2A
+MUSIC_WIND_MARIMBA                      equ $2B
+MUSIC_CORAL_TRIANGLE                    equ $2C
+MUSIC_ORGAN_OF_EVENING_CALM             equ $2D
+MUSIC_THUNDER_DRUM                      equ $2E
+MUSIC_MARIN_SING                        equ $2F
+MUSIC_MANBOS_MAMBO                      equ $30
+MUSIC_OVERWORLD_INTRO                   equ $31 ; After obtaining sword
+MUSIC_MR_WRITE_HOUSE                    equ $32
+MUSIC_ULRIRA                            equ $33
+MUSIC_TARIN_BEES                        equ $34
+MUSIC_FROG_SONG                         equ $35
```

Comment:
If you want to find this constant you can no longer search for "Mamu" to find it. That's unfair to you, though, since you couldn't find the old one if you searched for "Frog". Maybe a comment here mentioning Mamu could be useful?